mne/viz/utils.py

```diff
@@ -2153,7 +2155,7 @@ def _connection_line(x, fig, sourceax, targetax, y=1.,
                   clip_on=False)
 
 
-class DraggableLine:
+class DraggableLine(object):
```

Comment:
Is this really still necessary? I mean explicitly deriving from `object`?

Reply:
https://stackoverflow.com/questions/4015417/python-class-inherits-object

Reply:
Thanks - I don't even want to know, just really looking forward to 2020 :smile:.